### PR TITLE
set timeout geocoder

### PR DESCRIPTION
**Pull request by HugoRaza**
Set a timeout to geocode.

### CODE REVIEW

File: app/models/participation.rb [hunk 1]
```diff
@@ -2,5 +2,16 @@ class Participation < ApplicationRecord
   belongs_to :user
   belongs_to :playground
 
-  validates :status, inclusion: { in: %w[coming ongoing finished] }
```
Comment (aymericvinotperso) [inline app/models/participation.rb]: you should migrate the model motherfucker

File: app/views/playgrounds/show.html.erb [hunk 2]
```diff
@@ -10,7 +10,7 @@
   <%= participation.user.nickname  %>
   <%= participation.start_date  %>
   <%= participation.end_date  %>
-  <%= participation.status  %>
+  <%= participation.define_status  %>
```
Comment (aymericvinotperso) [inline app/views/playgrounds/show.html.erb]: hey man you should rather call it "status" right ?

Comment (HugoRaza) [reply app/views/playgrounds/show.html.erb]: Need also to change it in the model.

File: app/models/participation.rb (hunk 1 above)
Comment (HugoRaza) [reply app/models/participation.rb]: I'm working on it ! Wait for my next pull request.

**Pull request merged**